_includes/get-started-code-steps.html

```diff
@@ -0,0 +1,91 @@
+
+<div class="full-width-bg component-slim get-started-steps">
+  <div class="grid-wrapper">
+    <div class="width-2-12 width-12-12-m">
+      <h2 class="step">Step <span class="step-no">1</span></h2>
+    </div>
+    <div class="width-10-12 width-12-12-m">
+      <h3>Install via Command Line Interface</h3>
+      <p>Open your favorite CLI and paste this script to spin-up Quarkus.</p>
+      <h4>For Linux, macOS, and Windows (using WSL or bash compatible shell like cygwin or mingw)</h4>
+          <div class= "highlighter-rouge">
+            <div class="highlight">
+              <pre class="highlight"><code>
+curl -Ls https://sh.jbang.dev | bash -s - app install --fresh --force quarkus@quarkusio
+                 </code></pre>
```

Comment:
```suggestion
              <pre class="highlight"><code>curl -Ls https://sh.jbang.dev | bash -s - app install --fresh --force quarkus@quarkusio</code></pre>
```

all the code sections should not have start/end empty lines as that result in "copy" button giving you 3 lines instead of 1 which most terminals complains about.